src/test/scala/com/gd/twitteranalytics/TwitterReportGeneratorTest.scala

```diff
@@ -11,7 +11,7 @@ class TwitterReportGeneratorTest extends FreeSpec with BeforeAndAfter with DataF
 
   "TwitterReportGenerator " - {
     "getReportWithDataFrameProcessing" - {
-      "should generate report with users that are occuring more than 10 times per location" in {
+      "should generate report with maximum of 5 users that are occuring more than 10 times per location" in {
```

Comment:
the original requirement is finding top 5, not maximum 5. that makes difference.

Reply:
Fixed..